### PR TITLE
Crossport OVA-Util, VM-Util, and Util from master

### PR DESCRIPTION
**Pull request by zjs**
There have been a large number of tangled commits to master that modify various sets of robot resources and tests. The right thing to do would be to tease those apart and identify the pieces of each which are needed to fix deployment of the OVA on 1.4.1. Doing so has proved difficult, so this change crossports the current state of three key files.

This will very likely break nightly tests on the release branch in a variety of ways. This is acceptable because we don't currently run nightly tests on the release branch.

### CODE REVIEW

File: tests/common-ova/OVA-Cleanup.robot [hunk 1]
```diff
@@ -27,4 +27,5 @@ Copy OVA Support Bundle
     Copy Support Bundle  %{OVA_IP}
 
 Teardown Common OVA
-    Cleanup VIC Product OVA  %{OVA_NAME}
+    Log  Skipping cleanup of %{OVA_IP}
```
Comment (hickeng) [inline tests/common-ova/OVA-Cleanup.robot]: Don't merge.

**Comment by zjs** [comment]
Successful CI run: https://ci-vic.vmware.com/vmware/vic-product/5412

File: tests/resources/OVA-Util.robot [hunk 2]
```diff
@@ -113,17 +150,34 @@ Cleanup VIC Product OVA
...
+    # check for optional env variables
+    ${status}  ${message}=  Run Keyword And Ignore Error  Environment Variable Should Be Set  EXTERNAL_PSC
+    Run Keyword If  '${status}' == 'FAIL'  Set Environment Variable  EXTERNAL_PSC  ''
+    ${status}  ${message}=  Run Keyword And Ignore Error  Environment Variable Should Be Set  PSC_DOMAIN
+    Run Keyword If  '${status}' == 'FAIL'  Set Environment Variable  PSC_DOMAIN  ''
+
+    Log To Console  \nInitializing VIC appliance by API when API is available
     :FOR  ${i}  IN RANGE  30
-    \   ${rc}  ${out}=  Run And Return Rc And Output  curl -k -w "\%{http_code}\\n" --header "Content-Type: application/json" -X POST --data '{"target":"%{TEST_URL}:443","user":"%{TEST_USERNAME}","password":"%{TEST_PASSWORD}"}' https://${ova-ip}:9443/register
+    \   ${rc}  ${out}=  Run And Return Rc And Output  curl -k -w "\%{http_code}\\n" --header "Content-Type: application/json" -X POST --data '{"target":"%{TEST_URL}:443","user":"%{TEST_USERNAME}","password":"%{TEST_PASSWORD}","externalpsc":"%{EXTERNAL_PSC}","pscdomain":"%{PSC_DOMAIN}"}' https://${ova-ip}:9443/register
```
Comment (hickeng) [inline tests/resources/OVA-Util.robot]: we might need a `-s` here to ensure we only get the http_code response - I'm unsure.

Comment (zjs) [reply tests/resources/OVA-Util.robot]: Filed #1845 for this

**Pull request merged**